v2/test/postgresql_test.go

```diff
@@ -0,0 +1,377 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package test
+
+import (
+	"testing"
+	"time"
+
+	_ "github.com/jackc/pgx/v5/stdlib" //the pgx lib
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+
+	postgresqlv1 "github.com/Azure/azure-service-operator/v2/api/dbforpostgresql/v1"
+	postgresql "github.com/Azure/azure-service-operator/v2/api/dbforpostgresql/v1api20210601"
+	resources "github.com/Azure/azure-service-operator/v2/api/resources/v1api20200601"
+	"github.com/Azure/azure-service-operator/v2/internal/set"
+	"github.com/Azure/azure-service-operator/v2/internal/testcommon"
+	postgresqlutil "github.com/Azure/azure-service-operator/v2/internal/util/postgresql"
+	"github.com/Azure/azure-service-operator/v2/internal/util/to"
+	"github.com/Azure/azure-service-operator/v2/pkg/genruntime"
+)
+
+func Test_PostgreSql_Combined(t *testing.T) {
+	t.Parallel()
+	tc := globalTestContext.ForTest(t)
+
+	rg := tc.CreateTestResourceGroupAndWait()
+
+	adminUsername := "myadmin"
+	adminPasswordKey := "adminPassword"
+	adminPassword := tc.Namer.GeneratePassword()
+	secret := newPostgresSqlSecret(tc, adminPasswordKey, adminPassword)
```

Comment:
very minor: caps `SQL` rather than `Sql` is our standard style. Applies throughout this file

Reply:
Changed